ui/vite.config.js

```diff
@@ -0,0 +1,16 @@
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react";
+import svgr from "vite-plugin-svgr";
```

Comment:
we don't need this yet (and uninstall vite-plugin-svgr)

```suggestion
import svgr from "vite-plugin-svgr";
```